lib/cdo/github.rb

```diff
@@ -130,7 +130,7 @@ def self.open_pull_request_in_browser(base:, head:, title:)
       "?expand=1&title=#{CGI.escape title}"
   end
 
-  private_class_method def self.open_url(url)
+  def self.open_url(url)
```

Comment:
Though less than ideal, this seems like a reasonable hack as I expect this method to disappear when we move from `create_pull_request` to `create_and_merge_pull_request` for the DTP.